### DISCUSSION TITLE
Module  import CacheConfig in config.py

### DISCUSSION THREAD

**Discussion by dineshp89** [apache/superset#19335] [Q&A / Help] [answered]
Import error in config.py for CacheConfig

#### How to reproduce the bug

1. Open the config.py
2. Go to line number 48  in config.py
3. See "from superset.superset_typing import CacheConfig"
4. and try to "superset db upgrade"

### Expected results

Successful superset db upgrade 

### Actual results

Got below error:
ModulNotFoundError: No module Named 'superset.superset_typing'

### Environment

(please complete the following information):

- browser type and version:
- superset version: `1.4.1
- python version: `3.8`

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [ ] I have reproduced the issue with at least the latest released version of superset.
- [ ] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by dineshp89** [comment, marked as answer]
Please use 
'from superset.typing import CacheConfig' instead of 'from superset.superset_typing import CacheConfig'

and do  "superset db upgrade"

It will work.

**Comment by villebro** [comment]
`superset.typing` was recently renamed to `superset.superset_typing`. If you're running `docker-compose` you may need to do `docker-compose pull` for it to work.

**Reply by dineshp89** [reply]
@villebro , If we are using (superset.superset_typing), superset db upgrade failing. 

Redhat version 8.5 used.

**Comment by srinify** [comment]
Moving to Discussion so the debugging help can continue 🙏 If there's a clear bug you can help the community reproduce, then definitely open a Github Issue!
